test-kotest5/build.gradle

```diff
@@ -50,5 +57,5 @@ tasks.named("test") {
     }
     // PTS fails for Kotest because it's not supported, yet.
     // see https://docs.gradle.com/enterprise/predictive-test-selection/#_frameworks_and_languages
-    develocity.predictiveTestSelection.enabled = false
+//    predictiveSelection.enabled = false
```

Comment:
Do you want to keep that commented out? The `develocity.predictiveTestSelection.enabled=false` thing is the right syntax.